R/data.R

```diff
@@ -64,3 +62,157 @@
+#'   \item \code{proportion_diverted}: More details at \code{\link[DSMflow]{proportion_diverted}}
+#'   \item \code{total_diverted}: More details at \code{\link[DSMflow]{total_diverted}}
+#'   \item \code{delta_proportion_diverted}: More details at \code{\link[DSMflow]{delta_proportion_diverted}}
+#'   \item \code{delta_total_diverted}: More details at \code{\link[DSMflow]{delta_total_diverted}}
+#'   \item \code{weeks_flooded}: More details at \code{\link[DSMhabitat]{weeks_flooded}}
+#'   \item \code{prop_high_predation}: More details at \code{\link[DSMhabitat]{prop_high_predation}}
+#'   \item \code{contact_points}: More details at \code{\link[DSMhabitat]{contact_points}}
+#'   \item \code{delta_contact_points}: More details at \code{\link[DSMhabitat]{delta_contact_points}}
+#'   \item \code{delta_prop_high_predation}: More details at \code{\link[DSMhabitat]{delta_prop_high_predation}}
+#'   \item \code{..surv_juv_rear_int}: Intercept, Source: calibration (varies by tributary)
+#'   \item \code{..surv_juv_rear_contact_points}: Coefficient for \code{contact_points} variable, Source: calibration
+#'   \item \code{..surv_juv_rear_prop_diversions}: Coefficient for \code{prop_diversions} variable, Source: calibration
+#'   \item \code{..surv_juv_rear_total_diversions}: Coefficient for \code{total_diversions} variable, Source: calibration
+#'   \item \code{..surv_juv_bypass_int}: Intercept, Source: calibration
+#'   \item \code{..surv_juv_delta_int}: Intercept, Source: calibration
+#'   \item \code{..surv_juv_delta_contact_points}: Coefficient for \code{contact_points} variable, Source: calibration
+#'   \item \code{..surv_juv_delta_total_diverted}: Coefficient for \code{total_diversions} variable, Source: calibration
+#'   \item \code{.surv_juv_rear_avg_temp_thresh}:  Coefficient for \code{avg_temp_thresh} variable, Source: \href{https://www.tandfonline.com/doi/full/10.1577/M07-130.1?scroll=top&needAccess=true}{Runge et al (2008)}
+#'   \item \code{.surv_juv_rear_high_predation}: Coefficient for \code{high_predation} variable, source: \href{https://pubag.nal.usda.gov/catalog/512123}{Cavallo et al. (2012)}
+#'   \item \code{.surv_juv_rear_stranded}: Coefficient for \code{stranded} variable, source: \href{https://dsm-docs.s3-us-west-2.amazonaws.com/DWR-1140+USFWS+2006.pdf}{USFWS (2006) and CDWR (2006)}
+#'   \item \code{.surv_juv_rear_medium}: Parameter for medium sized fish, source: \href{https://afspubs.onlinelibrary.wiley.com/doi/abs/10.1577/M02-161.1}{Connor et al. (2004)}
+#'   \item \code{.surv_juv_rear_large}: parameter for large sized fish, source: \href{https://afspubs.onlinelibrary.wiley.com/doi/abs/10.1577/M02-161.1}{Connor et al. (2004)}
+#'   \item \code{.surv_juv_rear_floodplain}: parameter for floodplain rearing benefit, source: \href{https://dsm-docs.s3-us-west-2.amazonaws.com/SOMMER_T-SDWA+180+Floodplain+rearing+of+juvenile+chinook+salmon+evidence+of+enhanced+growth+and+survival+.pdf}{Sommer et al. (2001)}
```

Comment:
maybe break this section up into 3 lists "Tributary", "Bypass", and "Delta" all under the section "Rearing Survival"? If there is a parameter used by multiple rearing survival functions, have it listed in each spot with the source described in one and then with "See [blank] for source description" or something like that. make sense?

Reply:
Sorted into groups.